sdk/monitor/azure-monitor-ingestion/src/main/java/com/azure/monitor/ingestion/package-info.java

```diff
@@ -2,6 +2,56 @@
 // Licensed under the MIT License.
 
 /**
- *  Package containing client classes for uploading logs to Azure Monitor.
+ * The Azure Monitor Ingestion client library is used to send custom logs to an [Azure Monitor](https://learn.microsoft.com/azure/azure-monitor/overview)
+ * Log Analytics workspace.
+ *
+ * <h2>Getting Started</h2>
+ *
+ * <p><strong>Create the client</strong></p>
+ * An authenticated client is required to upload logs to Azure Monitor Log Analytics workspace. This package includes both synchronous and
+ * asynchronous forms of the clients. Use {@link com.azure.monitor.ingestion.LogsIngestionClientBuilder LogsIngestionClientBuilder} to
+ * customize and create an instance of {@link com.azure.monitor.ingestion.LogsIngestionClient LogsIngestionClient} or
+ * {@link com.azure.monitor.ingestion.LogsIngestionAsyncClient LogsIngestionAsyncClient}.
+ *
+ * <p><strong>Authenticate the client</strong></p>
+ * <p>
```

Comment:
add new line between two paragraphs.
The text can start right after paragraph tag.